Dockerfile.glibc

```diff
@@ -3,13 +3,13 @@ FROM debian:bookworm-slim AS builder
 
 # install required packages
 RUN apt-get update -y && \
-     apt-get install -y curl build-essential mold
+     apt-get install -y curl build-essential mold pkg-config libssl-dev
 
 # set up Rust toolchain
 RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
 ENV PATH="/root/.cargo/bin:${PATH}"
-RUN rustup toolchain install nightly-2024-01-12
-RUN rustup default nightly-2024-01-12
+RUN rustup toolchain install nightly-2023-06-25
+RUN rustup default nightly-2023-06-25
```

Comment:
this is incorrect for dev branch?

Reply:
i think dev branch should have new `nightly-2024-01-12` version

Reply:
Yeah